week1/index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="ko">
+
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/svg+xml" href="favicon.svg" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css">
+  <title>🍔 햄식이를 도와줘 🍔 - 김형겸</title>
+</head>
+
+<body>
+  <header>
+    <h1>🍔 햄식이를 도와줘 🍔</h1>
+  </header>
+  <main>git
+    <aside class="sidebar">
+      <h2 class="sidebar-title">다른거 먹자</h2>
+      <div class="sidebar-button">
+        <button class="option">맥도날드</button>
+        <button class="option">버거킹</button>
+      </div>
+    </aside>
+    <section class="burger">
+      <article class="burger-card">
+        <img src="https://www.mcdelivery.co.kr/kr//static/1648609546423/assets/82/products/1301.png" alt="빅맥">
+        <div class="burger-contents">
+          <h2>빅맥</h2>
+          <div class="burger-price">5,400원</div>
+          <p class="burger-desc">빅맥 만원팩 돌려내라</p>
```

Comment:
```suggestion
        <div class="burger__contents">
          <h2>빅맥</h2>
          <div class="burger__price">5,400원</div>
          <p class="burger__desc">빅맥 만원팩 돌려내라</p>
```
이 부분도 '__'를 이용해서 element요소로 구분해주면 좋을 것 같아요!